src/logic/calculate.js

```diff
@@ -0,0 +1,13 @@
+export default function calculate(calculatorData, buttonName) {
+  let { total, next } = calculatorData;
+  const { operation } = calculatorData;
+
+  if (buttonName === '+/-') {
```

Comment:
### Heads-Up:
**we need to add more cases;** 
1. if the `buttonName` = `A/C`, we should return `0`
2. if the `buttonName`  = `.` we should return `0.`